### ISSUE TITLE
enquiry about VIPriors Object Detection challenge 

### ISSUE THREAD

**Issue by hongrui16** [VIPriors/vipriors-challenges-toolkit#8]
I am sorry that I am very confused about your instruction about setting up dataset.
You say there are 3 json files, i.e, vipriors-object-detection-train.json, vipriors-object-detection-val.json, and vipriors-object-detection-test.json,  in your given annotations folder.
![image](https://user-images.githubusercontent.com/9922473/86511673-eef41200-be2d-11ea-9f76-ea5b3f3a6710.png)

However,  in arrange_images.py, it only gives me a clear explanation about training dataset. I do not know what is going on with test-images, for evaluation during training or for generating submission results?
Besides,  I do not know how to do with vipriors-object-detection-val.json and vipriors-object-detection-test.json.
![image](https://user-images.githubusercontent.com/9922473/86511676-f74c4d00-be2d-11ea-8955-f66d5a8b248b.png)

**could you explain to me about which group of images is used for generating submission results, which one is for test and evaluation during training?**

Many thanks.

**Comment by rjbruin** [comment]
Thanks for the question. I responded to your email as well, but let me respond here too for the public.

The validation set (val.json) is meant as a test set for use during development. The test set (test.json) is used for generating the submission and uploading to the server. You can also use the validation set to train on if you want.

It is also possible to upload your results on the validation set to the "validation phase" on the server to check if your submission format works. This can prevent failed submissions on the "test phase" for which the total amount of allowed submissions is limited.

**Comment by hongrui16** [comment]
> Thanks for the question. I responded to your email as well, but let me respond here too for the public.
> 
> The validation set (val.json) is meant as a test set for use during development. The test set (test.json) is used for generating the submission and uploading to the server. You can also use the validation set to train on if you want.
> 
> It is also possible to upload your results on the validation set to the "validation phase" on the server to check if your submission format works. This can prevent failed submissions on the "test phase" for which the total amount of allowed submissions is limited.

many thanks.
